src/components/Calculator.js

```diff
@@ -0,0 +1,51 @@
+/** ****************************************
+ * Project: math-magicians
+ * File: Calculator.js
+ * Created: 3/28/23
+ * Author: Abdullah Al Mamun <mamun1214@gmail.com>
+ ****************************************** */
+import React from 'react';
+import Buttons from './Buttons';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="dispaly">
+        <input className="input" placeholder="0" />
+      </div>
+      <div className="buttons">
+        <div className="btn-row">
+          <div className="btn-col"><Buttons label="AC" /></div>
+          <div className="btn-col"><Buttons label="+/-" /></div>
+          <div className="btn-col"><Buttons label="%" /></div>
+          <div className="btn-col"><Buttons label="/" /></div>
+        </div>
```

Comment:
- [ ] Please add the correct division symbol `÷` to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/228422083-2b7ec4a5-1e53-4bd4-8e95-d6d9d9ab3883.png)  | ![image](https://user-images.githubusercontent.com/68796232/228422025-eeb96ee4-0005-4543-8b0b-fcced59f93ce.png)  |